### PR TITLE
Switch kubectl port-forward -> NodePort as an attempt to stabilize 4621 from quarantine

### PR DESCRIPTION
**Pull request by akalenyu**
Signed-off-by: Alex Kalenyuk <akalenyu@redhat.com>

**What this PR does / why we need it**:
Our functional tests are using `kubectl port-forward` to upload the images, which is not very reliable.
Since https://github.com/kubevirt/kubevirtci/pull/653, we have a port that is exposed from host -> VM;
We can look it up and set up a NodePort against it instead.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tests/storage/imageupload.go [hunk 1]
```diff
@@ -131,7 +149,7 @@ var _ = SIGDescribe("[Serial]ImageUpload", func() {
 	}
 
 	Context("[rook-ceph] Upload an image and start a VMI with PVC", func() {
-		DescribeTable("[QUARANTINE][test_id:4621] Should succeed", func(resource, targetName string, validateFunc func(string, string), deleteFunc func(string), startVM bool) {
```
Comment (fgimenez) [inline tests/storage/imageupload.go:134]: We should not remove the `QUARANTINE` label in the same PR that tries to stabilize the test. After the fixes have landed we need to verify that they actually make the test less flaky for a period of time in the periodics (which execute all the quarantined tests), see https://github.com/kubevirt/kubevirt/blob/main/docs/quarantine.md#getting-tests-out-of-quarantine for details.

**Review by fgimenez — CHANGES_REQUESTED** [review]

File: tests/storage/imageupload.go (hunk 1 above)
Comment (akalenyu) [reply tests/storage/imageupload.go]: Got it. Done, thank you

**Review by fgimenez — APPROVED** [review]
/lgtm

Thanks!

**Pull request merged**